nilearn/datasets.py

```diff
@@ -150,9 +165,9 @@ def _chunk_read_(response, local_file, chunk_size=8192, report_hook=None,
     try:
         total_size = int(total_size) + initial_size
     except Exception, e:
-        if verbose > 0:
+        if verbose > 1:
```

Comment:
This is from the "silencing" PR... I suggest reviewing that one before this one, otherwise the two issues will be intertwined and hard to complete.

`verbose=0` was outputting here, so I upped everything (including the default) to keep everything on separate `verbose` levels, but to make `verbose=0` be silent.